src/state/ido/hooks.ts

```diff
@@ -0,0 +1,284 @@
+import { UNI } from '../../constants/index'
```

Comment:
are we using this file `src/state/ido/hooks.ts` anywhere ? if not please remove it.

Reply:
Removed this because it's not used